astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,493 @@
+                    names=("ring", "pericenter", "ascending node"),
+                )
+
+                units_list = [None, u.deg, u.deg]
+                ringtable = table.QTable(ringtable, units=units_list)
+
+            # Saturn F-ring data
+            elif group.startswith("F Ring"):
+                lines = group.split("\n")
+                for line in lines:
+                    l = line.split(":")
+                    if "F Ring pericenter" in l[0]:
+                        peri = float(re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()"))
+                    elif "F Ring ascending node" in l[0]:
+                        ascn = float(l[1].strip(", \n"))
+                ringtable = table.QTable(
+                    [["F"], [peri], [ascn]],
+                    names=("ring", "pericenter", "ascending node"),
+                    units=(None, u.deg, u.deg),
+                )
+
+            # Neptune ring arcs data
+            elif group.startswith("Courage"):
+                lines = group.split("\n")
+                for i in range(len(lines)):
+                    l = lines[i].split(":")
+                    ring = l[0].split("longitude")[0].strip(", \n")
+                    [min_angle, max_angle] = [
+                        float(s.strip(", \n"))
+                        for s in re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()").split()
+                    ]
+                    if i == 0:
+                        ringtable = table.QTable(
+                            [[ring], [min_angle], [max_angle]],
+                            names=("ring", "min_angle", "max_angle"),
+                            units=(None, u.deg, u.deg),
+                        )
+                    else:
+                        ringtable.add_row([ring, min_angle*u.deg, max_angle*u.deg])
+
+            else:
+                pass
+
+        # do some cleanup from the parsing job
+        if ringtable is not None:
+            ringtable.add_index("ring")
+
+        bodytable = table.join(bodytable, bodytable2)  # concatenate minor body table
+        bodytable.add_index("Body")
+
+        systemtable["obs_time"] = Time(
+            obs_time, format="iso", scale="utc"
+        )  # add obs time to systemtable
+
+        return systemtable, bodytable, ringtable
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for managing parser calls
+
+        Parameters
+        ----------
+        self :  RingNodeClass instance
+        response : string
+            raw response from server
+
+        Returns
+        -------
+        systemtable : dict
+        bodytable : `astropy.Table`
+        ringtable : `astropy.Table`
+        """
+        self.last_response = response
+        try:
+            systemtable, bodytable, ringtable = self._parse_ringnode(response.text)
+        except Exception as ex:
+            try:
+                self._last_query.remove_cache_file(self.cache_location)
```

Comment:
Why do you do this? To disable caching remove the kwarg instead and pass cache=False to `self._request`.

Reply:
honestly I have no idea how to do this "right," I pretty much copied everything here from the JPL Horizons query.  May I request some help here? I'm happy with however this ends up

Reply:
OK, let's come back to this once you push the updates. Without fixing it, at least in the mocks, the test will fail.

Reply:
And I was following the SIMBAD example 😄 😵‍💫 I'll look at a few updates to jplhorizons now.